surfactant/cmd/plugin.py

```diff
@@ -14,3 +15,74 @@ def plugin_list_cmd():
         print(f"canonical name: {pm.get_canonical_name(p)}")
         plugin_name = pm.get_name(p) if pm.get_name(p) else ""
         print(f"name: {plugin_name}")
+
+    # List disabled pluginsi
+    config_manager = ConfigManager()
+    section = "core"
+    section_key = "disable_plugins"
+
+    # Retrieve the current list of plugins that should be blocked
+    current_blocked_plugins = config_manager.get(section, section_key, [])
+
+    print("-------")
+    print("DISABLED PLUGINS")
+    for disabled_plugin in current_blocked_plugins:
+        print("-------")
+        print(f"name: {disabled_plugin}")
```

Comment:
```suggestion
    for disabled_plugin in current_blocked_plugins:
        print(f"\tname: {disabled_plugin}")
```
I think we can make the output a bit easier to follow with indentation